Clientinfo.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+
+namespace DNWS
+{
+  class Clientinfo : IPlugin
```

Comment:
1. Please follow the style guideline, i.e., all class name should be in the title case, e.g., ClientInfoPlugin
2. Please follow the pattern, i.e., put Plugin as suffix to all plugin class.